README.md

```diff
@@ -208,14 +231,18 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://www.scribbbles.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184864116-17978503-0ea3-48ca-a51a-fa1d634b4a9f.png' alt="Scribbles_free_illustrations" /></a> | [Scribbles](https://www.scribbbles.design/)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• SVG<br>• AI<br><br> ✌️Author <br> Designjoy                                                                       |
 | <a href="https://sketchvalley.com/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808722-b1fbf655-d2f9-4591-aad9-19c741a0bc78.png' alt="Sketchvalley_free_illustrations" /></a> | [Sketchvalley](https://sketchvalley.com/)  <br><br> ✔ Free <br><br>Format <br>• PNG<br>• SVG<br>• AI<br><br> |
 | <a href="https://weareskribbl.com"><img width='700' src='https://user-images.githubusercontent.com/74038190/184471692-07bd0ae8-4306-4cc1-bc1d-fcbed560f62c.png' alt="skribbl_free_illustrations" /></a> | [Skribbl](https://weareskribbl.com)  <br><br> ✔ Free <br><br>Format: <br> • PNG<br>• SVG<br> |
-| <a href="https://craftwork.design/downloads/slam/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184642751-2feaaab9-19bd-4e98-8dac-9560e3103516.png' alt="Slam_free_illustrations" /></a> | [Slam](https://craftwork.design/downloads/slam/)  <br><br> ✔ Free <br><br>Format <br> • Cinema 4D<br>• PNG<br><br> ✌️Author <br> Craftwork                                                                               |
+| <a href="https://craftwork.design/downloads/slam/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184642751-2feaaab9-19bd-4e98-8dac-9560e3103516.png' alt="Slam_free_illustrations" /></a> | [Slam](https://craftwork.design/downloads/slam/)  <br><br> ✔ Free <br><br>Format <br> • Cinema 4D<br>• PNG<br><br> ✌️Author <br> Craftwork  
+| <a href="https://drawkit.com/product/social-work-charity-illustrations"><img width='700' src='https://user-images.githubusercontent.com/34191321/194714455-c9e159b0-0e89-4698-9940-2eb3af67fa0a.png' alt="social_work_&_charity_free_illustrations" /></a> | [Social Work & Charity](https://drawkit.com/product/social-work-charity-illustrations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit |
```

Comment:
- change the src link to `https://user-images.githubusercontent.com/74038190/194873627-62964db3-9a06-4f7f-b184-19fab9d40a64.png` in Social Work & Charity
- add format `AI`